daprdocs/content/en/reference/components-reference/supported-state-stores/setup-cockroachdb.md

```diff
@@ -0,0 +1,74 @@
+---
+type: docs
+title: "CockroachDB"
+linkTitle: "CockroachDB"
+description: Detailed information on the CockroachDB state store component
+aliases:
+  - "/operations/components/setup-state-store/supported-state-stores/setup-cockroachdb/"
+---
+
+## Create a Dapr component
+
+Create a file called `cockroachdb.yaml`, paste the following and replace the `<CONNECTION STRING>` value with your connection string. The connection string for CockroachDB follow the same standard for PostgreSQL connection string. For example, `"host=localhost user=root port=26257 connect_timeout=10 database=dapr_test"`. See the CockroachDB [documentation on database connections](https://www.cockroachlabs.com/docs/stable/connect-to-the-database.html) for information on how to define a connection string.
+
+If you want to also configure CockroachDB to store actors, add the `actorStateStore` configuration element shown below.
```

Comment:
Given that the overview describes that this state store supports actors, you do not need to repeat this here. Suggest removing the actor examples.

Reply:
Of course, I'll make the adjustment 😄